server/views/pages/reviewPlan/reviewNote/index.njk

```diff
@@ -12,6 +12,20 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Add a note to this review</h1>
+        {{ govukWarningText({
+          text: "This note will not be sent to case notes",
+          iconFallbackText: "Warning"
+        }) }}
+        <p class="govuk-body">To help allocations, referrals and personal development include any details of:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>reviewing pathways based on skills, aspirations, and the activities available in your prison if the prisoner has transferred</li>
+          <li>helping them develop their CV and disclosure letters</li>
+          <li>referring and signposting them to other organisations to find employment - specify which ones</li>
+          <li>supporting the learning and skills manager and prison employment lead</li>
+          <li>any CIAG programme content provided - specify the type and subject, like job fairs, recruitment events, mock interviews, employer presentations (for example on application tips or career and sector awareness)</li>
+        </ul>
+        <br>
```

Comment:
Rather than a `<br>`, can we use one of the govuk margin styles on the `<ul>` element? Might be a bit neater 🤔